### PR TITLE
replace `%svelte.body%` etc with `%sveltekit.body%`

### PR DESCRIPTION
**Pull request by Rich-Harris**
Never got round to creating an issue for this, but I think it's a no-brainer — we should have been more explicit from the start that `%namespace.body%` is a SvelteKit thing, not a Svelte thing

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [x] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: .changeset/olive-lies-train.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'create-svelte': patch
+'@sveltejs/kit': patch
+---
+
+[breaking] Replace %svelte.body% etc with %sveltekit.body%
```
Comment (benmccann) [inline .changeset/olive-lies-train.md:6]: The etc makes more sense to me at the end
```suggestion
[breaking] Replace `%svelte.body%` with `%sveltekit.body%`, etc.
```

**Review by benmccann — APPROVED** [review]
I saw this the other day and was also thinking it's inconsistent with what we've done elsewhere

**Comment by Mlocik97** [comment]
I find it unnecesary long... can't we just have `%kit.body%` ? shorter to write if someone will write their own custom template or modify it.

**Comment by tcc-sejohnson** [comment]
@Rich-Harris

While we're at it, the root context for `$app/stores` is called `__svelte__` ([see here](https://github.com/sveltejs/kit/blob/master/packages/kit/src/runtime/app/stores.js)). Seems like renaming it to `__sveltekit__` would fit well with this PR?

**Comment by Rich-Harris** [comment]
We can make the `__svelte__` change separately since it's internal rather than external — it would probably be better to use a private object as the key rather than a magic string, but I don't know off the top of my head if that'd be a rabbit hole

**Pull request merged**